fixtures/appliance.py

```diff
@@ -26,15 +27,19 @@ def temp_appliances(count=1, preconfigured=True, lease_time=180):
         preconfigured: True if the appliance should be already configured, False otherwise
         lease_time: Lease time in minutes (3 hours by default)
     """
+    apps = None
     try:
         sprout_client = SproutClient.from_config()
         apps, request_id = sprout_client.provision_appliances(
             count=count, lease_time=lease_time, preconfigured=preconfigured)
         yield apps
+    except Exception as e:
+        logger.info('Exception detected: %s', str(e))
```

Comment:
I would recommend going for just `logger.exception('Error detected during appliance provision')` ... the traceback will get attached automatically.